db/schema.rb

```diff
@@ -96,8 +96,8 @@
     t.text "embedded_quiz_form_data"
```

Comment:
Could you remove your changes from this file?

Reply:
Fixed